src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/oops/ConstantPool.java

```diff
@@ -125,9 +121,9 @@ public ConstantPool(Address addr) {
   private static long headerSize;
   private static long elementSize;
 
-  private static int INDY_BSM_OFFSET;
-  private static int INDY_ARGC_OFFSET;
-  private static int INDY_ARGV_OFFSET;
+  private static int INDY_BSM_OFFSET = 0;
+  private static int INDY_ARGC_OFFSET = 1;
+  private static int INDY_ARGV_OFFSET = 2;
```

Comment:
Is the SA code that references these fields still correct? It seems the references are from the ClassWriter, which we don't have very good test coverage for. Probably this is a bug that is not being detected.

Reply:
The references are still correct, as we haven't changed the struct layout. We're also not planning on doing so. I don't think that there is a bug here.

Reply:
Ok. It's just unusual to see constants like these that are needed by SA but serve no purpose in hotspot.